### PR TITLE
Add standalone npd on GCI.

### PR DESCRIPTION
**Pull request by Random-Liu**
This PR added standalone NPD in GCE GCI cluster. I already verified the PR, and it should work.

/cc @dchen1107 @fabioy @andyxning @kubernetes/sig-node-misc 

**Release note**:

```release-note
Node Problem Detector is beta now. New features added: journald support, standalone mode and arbitrary system log monitoring.
```

### CODE REVIEW

File: plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -298,6 +298,8 @@ func ClusterRoleBindings() []rbac.ClusterRoleBinding {
 		rbac.NewClusterBinding("system:node").Groups(user.NodesGroup).BindingOrDie(),
 		rbac.NewClusterBinding("system:node-proxier").Users(user.KubeProxy).BindingOrDie(),
 		rbac.NewClusterBinding("system:kube-controller-manager").Users(user.KubeControllerManager).BindingOrDie(),
+		// FIXME: Define me in yaml.
+		rbac.NewClusterBinding("system:node-problem-detector").Users(user.NodeProblemDetector).BindingOrDie(),
```
Comment (liggitt) [inline plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go]: maybe something more like https://github.com/kubernetes/kubernetes/pull/40760/files#diff-6e3b476b9225d1213dc6ad13e453fc16R1138 ?

**Comment by Random-Liu** [comment]
@dchen1107 This PR is ready for review. But note that before merging this PR, we need to merge all pending PRs in npd, and release the v0.3.0-alpha.1 version of NPD.

File: plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go (hunk 1 above)
Comment (Random-Liu) [inline plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go]: Done. :)

**Comment by dchen1107** [comment]
Please update your pr description above:

>However, there are still dependencies need to be resolved before merging this PR:
> Figure out how to do role binding for standalone NPD. (Ref #40021)

I believe the issue is resolved in this pr.

> Get NPD journald support PR merged, build a release tar ball, and update the tar version and path in this PR.

I believe Those prs are merged.

**Comment by dchen1107** [comment]
/lgtm 
and 
/approve

**Comment by Random-Liu** [comment]
Apply LGTM based on https://github.com/kubernetes/kubernetes/pull/40206#issuecomment-280811883.

**Pull request merged**